src/test/Components.test.js

```diff
@@ -0,0 +1,122 @@
+import {InputForm} from "../components/InputForm";
+import {ADD, END, START} from "../constants";
+import {addCat} from "../actions";
+
+configure({adapter: new Adapter()});
+
+
+describe("Counter", () => {
+    it("should render counter", () => {
+        const wrapper = shallow(<Counter count={1}/>);
+        expect(wrapper).toMatchSnapshot();
+    });
+});
+
+describe("Delete button render", () => {
+    it("should render Delete button", () => {
+        const deleteImages = jest.fn();
+        const wrapper = shallow(<DeleteButton countToDelete={1} deleteImages={deleteImages}/>);
+        expect(wrapper).toMatchSnapshot();
+    });
+});
+
+describe("Delete button onClick", () => {
+    it("should handle onClick in Delete button", () => {
+        const deleteImages = jest.fn();
+        const wrapper = shallow(<DeleteButton countToDelete={1} deleteImages={deleteImages}/>);
+        wrapper.find("button").simulate("click")
+        expect(deleteImages).toHaveBeenCalled();
+    });
+});
+
+describe("Image list", () => {
+    it("should render ImageList", () => {
+        const wrapper = shallow(<ImageList images={[]}/>);
+        expect(wrapper).toMatchSnapshot();
+    });
+});
+
+describe("Image list container render", () => {
+    it("should render ImageListContainer", () => {
+            const addImagesToStore = jest.fn();
+            const wrapper = shallow(<ImageListContainer images={[]} addImagesToStore={addImagesToStore}/>);
+            expect(wrapper).toMatchSnapshot();
+        }
+    )
+    ;
+});
+
+describe("Image list container handle scroll", () => {
+    it("should handle scroll to bottom in ImageListContainer", () => {
+            const addImagesToStore = jest.fn();
+            const wrapper = shallow(<ImageListContainer images={[]} addImagesToStore={addImagesToStore}/>);
+            const event = {
+                target: {
+                    scrollingElement: {
+                        scrollHeight: 0,
+                        clientHeight: 0,
+                        scrollTop: 0,
+                    }
+                }
+            }
+            wrapper.simulate('scroll', event)
+            expect(addImagesToStore).toHaveBeenCalled();
+        }
+    )
+    ;
+});
+
+describe("Input Form render", () => {
+    it("should render InputForm", () => {
+        const wrapper = shallow(<InputForm countToDelete={1} updateCountToDelete={jest.fn}/>);
+        expect(wrapper).toMatchSnapshot();
+    });
+});
+
+describe("Input Form onChange", () => {
+    it("should handle onChange in InputForm", () => {
+        const updateCountToDelete = jest.fn()
+        const wrapper = shallow(<InputForm class="a" countToDelete={1} updateCountToDelete={updateCountToDelete}/>);
+        const event = {target: {value: 2}}
+        wrapper.find("input").simulate('change', event)
+        expect(updateCountToDelete).toHaveBeenCalled()
+
+    });
+});
+
+describe('Test thunk action when is not fetching', () => {
```

Comment:
It's a common practice in Venom to have a single describe for all the tests related to the same function/component.

Reply:
No need to fix the tests in this PR, just FYI.